src/app/bulk-edit/bulk-edit-select/bulk-edit-select.component.ts

```diff
@@ -61,8 +61,8 @@ interface CatalogueItemDomainTypeOption {
   styleUrls: ['./bulk-edit-select.component.scss']
 })
 export class BulkEditSelectComponent implements OnInit, OnDestroy {
-  @Output() onCancel = new EventEmitter<void>();
-  @Output() onNext = new EventEmitter<void>();
+  @Output() cancel = new EventEmitter<void>();
+  @Output() next = new EventEmitter<void>();
```

Comment:
The output bindings have been changed in these child controls, but the parent components that use these have not. Currently, when testing the bulk editor, I cannot progress forward or backward after making selections. The `mdm-bulk-edit-container` component also needs to be updated:

```html
<!-- Current usage of the components, using wrong output names -->
<mdm-bulk-edit-select
  *ngIf="currentStep === Steps.Selection"
  [(context)]="context"
  (onNext)="next()"
  (onCancel)="cancel()"
></mdm-bulk-edit-select>
```

Reply:
Thank you - good catch.  It appears to be just that one instance that has an issue.  It has a binding for  `[(context)]` that doesn't appear to exist.  Shall I ditch it now, or leave it for another tidying session?